qutip/tests/core/data/test_reshape.py

```diff
@@ -0,0 +1,135 @@
+from .test_mathematics import UnaryOpMixin
+import pytest
+import numpy as np
+from qutip import data
+from qutip.core.data import CSR, Dense
+
+
+class TestSplitColumns(UnaryOpMixin):
+    def op_numpy(self, matrix):
+        return [matrix[:, i].reshape((-1, 1)) for i in range(matrix.shape[1])]
```

Comment:
Can also just be `return list(matrix.T)`.

Reply:
This seems to change the shape of each individual element in the list to be (2,) instead of (2,1).

Reply:
oh yeah, of course.  I guess it would have to be `[column[:, np.newaxis] for column in matrix.T]`, but then I'm not sure it's much better than what you've got already.

Reply:
I actually think is more readable.

Reply:
Up to you.  Yours is certainly fine to read as well - my original suggestion was only because I thought it was basically a trivial iterator, forgetting the extra shape dimension.